src/canisters/governance/services.ts

```diff
@@ -1,28 +1,45 @@
 import { GovernanceService } from "../../../candid/governance.idl";
-import { ManageNeuron } from "../../../candid/governanceTypes";
+import {
+  Command_1,
+  ManageNeuron,
+  ManageNeuronResponse,
+} from "../../../candid/governanceTypes";
 import { GovernanceError } from "../../errors/governance.errors";
 
 /**
+ * Checks a Manage Neuron Response for error and returns successful response data.
+ *
  * @throws {@link GovernanceError}
  */
-export const manageNeuron = async ({
-  request,
-  service,
-}: {
-  request: ManageNeuron;
-  service: GovernanceService;
-}): Promise<void> => {
-  const { command } = await service.manage_neuron(request);
-  const response = command[0];
-
-  if (!response) {
+export const getSuccessfulCommandFromResponse = (
```

Comment:
I would suggest to make this function `void` and named it `assertSuccessfulCommandFromResponse`

It would make more obvious what it does. The result of the function is not used everywhere so the return doesn't add much from what I read

Reply:
what do you think?

Reply:
It is used twice in the governance file.

I also thought of doing another function which only `assert`s, but I also need the `getCommand`. If I use the `assert` inside the `getCommand`, then I need to cast the command, because TS is not smart enough:

Something like the following:
```
export const getSuccessfulCommandFromResponse = (r) => {
  assert(r);
  return r.command[0] as Command_1;
}
```

What do you prefer?

Reply:
Let's go for your original PR solution. thanks for the feedback.